docs/ref/contrib/contenttypes.txt

```diff
@@ -344,14 +344,19 @@ remain set to their original values and the ``GenericForeignKey`` returns
 
 Due to the way :class:`~django.contrib.contenttypes.fields.GenericForeignKey`
 is implemented, you cannot use such fields directly with filters (``filter()``
-and ``exclude()``, for example) via the database API. Because a
-:class:`~django.contrib.contenttypes.fields.GenericForeignKey` isn't a
+and ``exclude()``, for example) for nested lookups via the database API.
+Because a :class:`~django.contrib.contenttypes.fields.GenericForeignKey` isn't a
 normal field object, these examples will *not* work::
 
     # This will fail
-    >>> TaggedItem.objects.filter(content_object=guido)
+    >>> TaggedItem.objects.filter(content_object__username='Guido')
     # This will also fail
+    >>> TaggedItem.objects.get(content_object__username='Guido')
+
+Simple lookups, however, will work:
```

Comment:
I think this bears a more substantial rewrite. Being able to filter using the object should come first, followed by the `Due to the way...` limitation.

Reply:
Rewrote the whole paragraph as per review